.github/ISSUE_TEMPLATE.md

```diff
@@ -1,15 +1,18 @@
 <!--
 Have you read Mocha's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/mochajs/mocha/blob/master/.github/CODE_OF_CONDUCT.md
+
 For more, check out the Mocha Gitter chat room: https://gitter.im/mochajs/mocha
 -->
 
 ### Prerequisites
 <!--
 Place an `x` between the square brackets on the lines below for every satisified prerequisite.
 -->
-* [ ] Checked that your issue isn't already filed by cross referencing [issues with the `common mistake` label](https://github.com/mochajs/mocha/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Acommon-mistake%20)
-* [ ] Checked next-gen ES issues and syntax problems by using the same environment and/or transpiler configuration without Mocha to ensure it isn't just a feature that actually isn't supported in the environment in question or a bug in your code.
-* [ ] 'Smoke tested' the code to be tested by running it outside the real test suite to get a better sense of whether the problem is in the code under test, your usage of Mocha, or Mocha itself
+* [ ] Checked that your issue isn't already filed by searching for relevant keywords in the [issue tracker](https://github.com/mochajs/mocha/issues) and cross-referencing [issues with the `common mistake` label](https://github.com/mochajs/mocha/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Acommon-mistake%20) <!-- NOTE: in the event that you find a relevant existing issue but its solution does not work for you, it is possible that you have a different although similar issue; in such cases opening a new issue is appropriate. ***However***, in the event you do wish to add to an existing issue (e.g. because it has no solution yet), **we highly recommend following the same steps and instructions as in reporting a new issue**, to minimize comments troubleshooting your particular case on someone else's issue. If you don't have new information to provide, just hit the "thumbs-up" reaction on the issue; we can look at those numbers to get an idea of how many people are affected without spam in the comments. -->
+* [ ] 'Smoke tested' the code to be tested by running it outside the real test suite to get a better sense of whether the problem is in the code under test, your usage of Mocha, or Mocha itself. If using any additional tooling make sure to use the exact same toolset outside Mocha (see below).
+* [ ] Checked next-gen ES issues and syntax problems (if any) by using ***the same*** *environment and/or transpiler configuration* ***without Mocha*** to ensure it isn't just a feature that actually isn't supported in the environment in question or a bug in your code. For example:
```

Comment:
like previously:

* * *

* [ ] Ensure your environment supports all JavaScript features you are attempting to use